api/context.go

```diff
@@ -52,6 +57,11 @@ func (r *Context) Attach(ctx *gin.Context) {
 // Detach from gin context
 func (r *Context) Detach() {
 	delete(r.Context.Keys, "RichContext")
+	atomic.AddUint64(&requestCount, 1)
+	if requestCount%100 == 0 {
```

Comment:
Just curious, was there some experiment behind the `%100` or well qualified expectation?